Tools/CMake/AMReX_Config.cmake

```diff
@@ -48,9 +48,14 @@ function (configure_amrex)
    # Moreover, it will also enforce such standard on all the consuming targets
    #
    set_target_properties(amrex PROPERTIES CXX_EXTENSIONS OFF)
-
    target_compile_features(amrex PUBLIC cxx_std_11)  # minimum: C++11
 
+   if (ENABLE_CUDA AND (CMAKE_VERSION VERSION_GREATER_EQUAL 3.17) )
```

Comment:
I added a check on the CMake version because CUDA meta-features were introduced with CMake 3.17, see: [https://blog.kitware.com/cmake-3-17-0-rc3-is-ready-for-testing/](https://blog.kitware.com/cmake-3-17-0-rc3-is-ready-for-testing/).